runtime/browser/xwalk_browser_context.cc

```diff
@@ -349,6 +315,17 @@ net::URLRequestContextGetter*
 #endif
 }
 
+net::URLRequestContextGetter* XWalkBrowserContext::CreateMediaRequestContext() {
```

Comment:
With https://codereview.chromium.org/1892893002 in mind:
* `GetMediaRequestContextForStoragePartition()` in `android_webview/` did not do anything in the past, so `AwBrowserContext::CreateMediaRequestContextForStoragePartition()` is now returning `NULL`.
* `chrome/`'s `ProfileImpl::GetMediaRequestContextForStoragePartition()`, on the other hand, had an actual implementation, which was kept in `ProfileImpl::CreateMediaRequestContextForStoragePartition()`.
* Analogously, since `XWalkBrowserContext::GetMediaRequestContextForStoragePartition()` was not empty, I don't see why `XWalkBrowserContext::CreateMediaRequestContextForStoragePartition()` should.